ts/src/kucoin.ts

```diff
@@ -2410,9 +2411,6 @@ export default class kucoin extends Exchange {
         if (cancelExist) {
             status = 'canceled';
         }
-        if (status === undefined) {
-            status = 'closed';
-        }
```

Comment:
@samgermain why are you removing this?

Reply:
https://github.com/ccxt/ccxt/pull/16674#discussion_r1092393759

Reply:
@samgermain I think that by removing this comment, we will stop returning the `closed` status but ok

Reply:
@carlosmiei well the "closed" status is wrong in this case - creating orders on kucoin returns only the ID - so this will trigger the ccxt to return "closed" instead (status is empty).

For market orders, closed may be fine, but it's wrong for limit orders. they **could** be closed - but they can just as well be open, so ccxt is returning the wrong response in this case.